package.json

```diff
@@ -1,21 +1,31 @@
 {
-  "name": "electrum-client",
-  "version": "0.0.7",
-  "description": "Electrum protocol client for node.js",
+  "name": "electrum-client-js",
+  "version": "0.1.0",
+  "description": "Electrum protocol client for JS",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha --timeout 10000",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
+  },
+  "dependencies": {
+    "websocket": "^1.0.29"
   },
-  "dependencies": {},
   "devDependencies": {
-    "electrum-host-parse": "*"
+    "chai": "^4.2.0",
+    "electrum-host-parse": "*",
```

Comment:
Add a version here, I see `^0.1.1` is latest

Reply:
Still unaddressed

Reply:
Yes, I'm not sure if we need this dependency at all, it's used only in examples. I have no decision on this yet.

Reply:
Version set.